libweb3jsonrpc/Debug.cpp

```diff
@@ -134,3 +137,34 @@ Json::Value Debug::debug_storageAt(string const& _blockHashOrNumber, int _txInde
 		ret[toHex(toCompactBigEndian(i.first, 1), 2, HexPrefix::Add)] = toHex(toCompactBigEndian(i.second, 1), 2, HexPrefix::Add);
 	return ret;
 }
+
+Json::Value Debug::debug_traceCall(Json::Value const& _call, std::string const& _blockNumber, Json::Value const& _options)
+{
+	Json::Value ret;
+	try
+	{
+		Block temp = m_eth.asOf(jsToBlockNumber(_blockNumber));
```

Comment:
this line need asOf to be public, now it is set as protected .... https://github.com/ethereum/libethereum/blob/develop/libethereum/Client.h#L213
The reason is that we usually gives the result of ' jsToBlockNumber(_blockNumber) ' directly to the client which take care of getting the block and processing it. in this case we don't have any debugging functions in the client interface and we have to process everything on Debug.cpp which is maybe not a good thing. maybe we should add traceCall an traceTransaction to the client.
Please let me know what do you think about that.